azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +34,16 @@
   QUEUED(110),
   FAILED_SUCCEEDED(120),
   CANCELLED(130);
+  // status is TINYINT and in H2 DB the possible values are: -128 to 127
```

Comment:
Yes, that's what I thought too. I was hoping someone smarter than me can come up with a better solution. :)